checkov/version.py

```diff
@@ -1 +1 @@
-version = '1.0.863'
```

Comment:
Hey @ebrucucen  can you uncommit this change? `version.py` is auto updated as part of ci/cd pipeline

Reply:
Sure, I have removed the change with a merge from the upstream @schosterbarak.